transports/plaintext/src/error.rs

```diff
@@ -28,7 +28,7 @@ pub enum PlainTextError {
     IoError(IoError),
 
     /// Failed to parse the handshake protobuf message.
-    InvalidPayload(Option<prost::DecodeError>),
+    InvalidPayload(Option<protobuf::Error>),
```

Comment:
This makes it a breaking change too I think.

Reply:
Yes, it's public.